src/AppContainer.js

```diff
@@ -1,10 +1,14 @@
 import App from "./App";
 import "./App.css";
 import { connect } from "react-redux";
+import {loadContacts, loadVehicles, loadComments, loadProducts} from "./actions";
 
 function mapDispatchToProps(dispatch) {
   return {
-
+    loadContacts,
```

Comment:
Try coding this the normal way. you need to dispatch the action, not just call it directly